mmdeploy/codebase/mmdet/models/dense_heads/yolo_head.py

```diff
@@ -130,18 +130,6 @@ def __mark_pred_maps(pred_maps):
 
     post_params = get_post_processing_params(ctx.cfg)
     score_threshold = cfg.get('score_thr', post_params.score_threshold)
```

Comment:
seems the line  `score_threshold = cfg.get('score_thr', post_params.score_threshold)` is also useless.